src/test/java/com/gizasystems/automationexercise/tests/VerifyingScrollUpUsingArrowButtonAndScrollDownTests.java

```diff
@@ -0,0 +1,47 @@
+package com.gizasystems.automationexercise.tests;
+
+import com.gizasystems.automationexercise.pages.Footer;
+import com.gizasystems.automationexercise.pages.HomePage;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.*;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+@Epic("Automation Exercise")
+@Feature("Home page navigation")
+@Story("Verify Scroll Up using 'Arrow' button and Scroll Down functionality")
+public class VerifyingScrollUpUsingArrowButtonAndScrollDownTests {
+    // Variables
+    private SHAFT.GUI.WebDriver driver;
+    private SHAFT.TestData.JSON testUser;
+    // Test Cases
```

Comment:
Please add empty line here between the variables and testcases sections.

Reply:
Done